sql/internet_outages/global_outages_v1/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Internet Outages
+description: |-
+  TODO
+owners:
+  - aplacitelli@mozilla.com
+  - sguha@mozilla.com
+labels:
+  incremental: true
+  public_json: true
```

Comment:
Just double checking, setting this to `true` will make the data public to everyone. Is that intended?

Reply:
Ah, darn, no, not intentional at all.